package main

func main() {
	err := do()

	if err != nil {
		//	handle err!
		//	terminate err!
	}

	//	success!
	//  continue...
}
